lib/rules/exports-style.js

```diff
@@ -139,6 +139,87 @@ function getExportsNodes(scope) {
     return variable.references.map(reference => reference.identifier)
 }
 
+function getReplacementForProperty(property, sourceCode) {
+    if (property.kind === "get" || property.type === "SpreadElement") {
+        // module.exports = { get foo() { ... } }
+        // module.exports = { foo }
+        // module.exports = { ...foo }
+        // We don't have a similarly nice syntax for adding these directly
+        // on the exports object. Give up on fixing the whole thing.
+        return undefined
```

Comment:
it's more common to return null in fixers.

```suggestion
        return null
```

Reply:
a4ec5000a715ec5c698e487bb685a9b2987b5a1f